en-US/firefox/products/developer-quantum.lang

```diff
@@ -210,10 +295,6 @@ Welcome to the all-new Firefox Quantum: Developer Edition
 Firefox has been rebuilt from the ground-up to be faster, sleeker, and more powerful than ever.
 
 
-;Firefox Devtools
-Firefox Devtools
```

Comment:
Please add this back. Remember that the errors page shows you if there are duplicate strings, and this is not (check the case).